src/libhirte/common/time-util.c

```diff
@@ -20,3 +22,25 @@ uint64_t finalize_time_interval_micros(int64_t start_time_micros) {
 double micros_to_millis(uint64_t time_micros) {
         return (double) time_micros * microsec_to_millisec_multiplier;
 }
+
+int get_formatted_timestamp(char *timebuf_ret, const size_t buf_size) {
+        struct timespec now;
+        int r = 0;
+        const size_t timestamp_size = 20;
+        char timebuf[timestamp_size];
+
+        if (buf_size < timestamp_size + 4) {
+                return -EINVAL;
+        }
+
+        r = clock_gettime(CLOCK_REALTIME, &now);
+        if (r < 0) {
+                return r;
+        }
+
+        strftime(timebuf, timestamp_size, "%Y.%m.%d %H:%M:%S", localtime(&now.tv_sec));
```

Comment:
Could you please use ISO 8601 date time format? Or at least it's more common variant, where date and time is separated by space?

https://datatracker.ietf.org/doc/html/rfc3339#section-5.6

So `YYYY-MM-DD hh:mm:ssZ` should correspond to `%F %T%z` format for strftime

Reply:
Narrowed it to time only.
Do we need this Z notation?

Reply:
If you will manage several machines and some of them them will have different timezone than the other, then showing the timezone is important

Reply:
I think the timezone and time should be relative to the clients timezone.

Reply:
I'd log complete timestamp including timezone anyway

Reply:
@mwperina both for `stderr` and `stderr-full`? Or just time for `stderr` and full timestamp for `stderr-full`?

Reply:
Both, so they are aligned

Reply:
I agree, makes sense to use the same timestamp format for both if added to `stderr`.